openfoodfacts_taxonomy_parser/parser/exception.py

```diff
@@ -0,0 +1,8 @@
+class FileError(ValueError):
```

Comment:
I will use Exception (we may want to catch it independently of ValueError which is very common, and I don't really see an obvious link.) My previous comment might as misled you !

Also I don't find FileError really descriptive, while you are raising in a very specific case. Maybe DuplicateIDError would be better. (or something better ?)

```suggestion
class DuplicateIDError(Exception):
```